index.bs

```diff
@@ -2203,6 +2210,14 @@ browsingContext.InnerTextLocator = {
    ? maxDepth: js-uint,
 }
 
+browsingContext.AccessibilityLocator = {
```

Comment:
nit: Please move up before `CssLocator` to keep the alphabetical order. Same for `browsingContext.Locator` as well.